src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -77,6 +77,39 @@ public async Task FishUntilMessageAsync<T>(TimeSpan? max = null)
             });
         }
 
+        /// <summary>
+        /// Waits for a <paramref name="max"/> period of 'radio-silence' a number of <paramref name="maxIterations"/> iterations.
+        /// Note: 'radio-silence' definition: period when no messages arrive at.
+        /// </summary>
+        /// <param name="max">A temporary period of 'radio-silence'.</param>
+        /// <param name="maxIterations">The method asserts that <paramref name="maxIterations"/> is never reached.</param>
+        /// If set to null then this method will loop for an infinite number of <paramref name="max"/> periods. 
+        /// NOTE: If set to null and radio-silence is never reached then this method will never return.  
+        /// <returns>Returns all the messages encountered before 'radio-silence' was reached.</returns>
+        public async Task<IReadOnlyList<object>> WaitForRadioSilenceAsync(TimeSpan? max = null, uint? maxIterations = null)
```

Comment:
So this is, in essence, to wait for message traffic to gradually settle down - distinct from `ExpectNoMessages` right?